drizzlepac/hlautils/catalog_utils.py

```diff
@@ -567,19 +567,29 @@ def identify_sources(self, **pars):
             log.info("{}".format("=" * 80))
 
             # find ALL the sources!!!
-            log.info("DAOStarFinder(fwhm={}, threshold={}*{})".format(source_fwhm, self.param_dict['nsigma'],
-                                                                      self.image.bkg_rms_median))
+            use_DaoStarFinder = True # Set to "True" to use DaoStarFinder (The default method), instead of IrafStarFinder to identifiy soruces
```

Comment:
It is not clear to me that the following 'if use_DaoStarFinder' can ever be False, which would make the if_else statement un-necessary.

Reply:
@stsci-hack , I'll add it as a new parameter in the "catalog_generation_all.json" parameter files.

Reply:
That would work just fine.

Reply:
Issue resolved in commit [ac6b7fc](https://github.com/spacetelescope/drizzlepac/pull/602/commits/ac6b7fc9e116b2ed5899219553e791d74558f092)